spec/src/main/asciidoc/core/injectionandresolution_full.asciidoc

```diff
@@ -70,49 +88,70 @@ The container automatically detects inconsistent specialization and treats it as
 
 ==== Inter-module injection in {cdi_full}
 
-In addition to rules defined in <<inter_module_injection>>, when running in {cdi_full}, the following apply:
+In addition to rules defined in <<inter_module_injection>>, the following rules apply.
 
 A bean is also _available for injection_ in a certain module if:
 
-* the bean is not a decorator
+* the bean is not a decorator,
+* the bean is either not an alternative, or the module is a bean archive and the bean is a selected alternative of the bean archive.
```

Comment:
+1 for overriden becuase with the last sentence you added, it becomes very cryptic to figure out the rules.

Reply:
I think a similarly cryptic sentence is present in the original, but I see what you mean :-)

Reply:
Yeah,... I never said the original was better :D